doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -60,6 +60,8 @@ Using the nRF54H20 DK with the |NCS| version |release| also requires the followi
 +----------------------------------------------+-------------------------------------+
 | nRF Util ``trace`` command                   | version |54H_nrfutil_trace_ver|     |
 +----------------------------------------------+-------------------------------------+
+| nRF Util ``sdk-manager`` command             |  Latest                             |
++----------------------------------------------+-------------------------------------+
```

Comment:
No, this is incorrect.
If you install NCS and its toolchain, you already have sdk-manager latest.
Remove this edit.